lectures/markov_chains_II.md

```diff
@@ -381,21 +343,11 @@ $$
 
 The diagram of the Markov chain shows that it is **irreducible**
```

Comment:
we use `**bold**` for definitions only, as discussed in the style manual.  Could you please change `**irreducible**` to `irreducible`?